### DISCUSSION TITLE
Implement node-globals-polyfill  without patching to support wallet connect [web3]

### DISCUSSION THREAD

**Discussion by mekunka** [remix-run/remix#3203] [Proposals]
Hi,

I'm to one who is trying to use remix with web3 wallets.
There is no problem with connecting with metamask but not with walletconnect and coinbase wallet.
I tried all other web3 wallet library but had no luck at all. Every library faces the same problem "**Buffer is not defined**"

Finally I found solution but with old version of remix and "wagmi" from this https://github.com/tmm/wagmi/pull/64
However, it's the patch with remix version 1.1.1, I saw latest version already implemented node-modules-polyfill but not node-globals-polyfill.
Is it possible to implement this is the package? Then we don't have to do any patch, I thin others also faced the same problem as well.

Thanks

**Comment by jdnichollsc** [comment]
@mekunka hello mate, did you find any example to add the **[Buffer](https://www.npmjs.com/package/buffer)** polyfill? I'm trying to do that from **root** and **entry.client** files without success, thanks for your help!
